.github/workflows/go-test.yaml

```diff
@@ -9,6 +9,19 @@ permissions:
   contents: read
 
 jobs:
+  uncommitted-code-check:
+    name: uncommitted-code-check
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
you need to checkout/clone the repository first

Reply:
Ah right, missed that.

Reply:
Shall I add the changes to the `release-*` branches as well?
Or, let it be done post the PR is cherry-picked?

Reply:
We can automatically cherry-pick it after merging.